src/components/Footer/Footer.stories.tsx

```diff
@@ -0,0 +1,301 @@
+    }
+    secondary={
+      <Logo
+        slim
+        image={
+          <img
+            className="usa-footer__logo-img"
+            alt="img alt text"
+            src={logoImg}
+          />
+        }
+        heading={<h3 className="usa-footer__logo-heading">Name of Agency</h3>}
+      />
+    }
+  />
+)
+
+export const MediumFooter = (): React.ReactElement => (
+  <Footer
+    medium
+    returnToTop={returnToTop}
+    primary={
+      <FooterNav
+        medium
+        links={Array(4).fill(
+          <a className="usa-footer__primary-link" href="#">
+            Primary Link
+          </a>
+        )}
+      />
+    }
+    secondary={
+      <div className="grid-row grid-gap">
+        <Logo
+          medium
+          image={
+            <img
+              className="usa-footer__logo-img"
+              alt="img alt text"
+              src={logoImg}
+            />
+          }
+          heading={<h3 className="usa-footer__logo-heading">Name of Agency</h3>}
+        />
+        <div className="usa-footer__contact-links mobile-lg:grid-col-6">
+          <SocialLinks
+            links={[
+              <a
+                key="facebook"
+                className="usa-social-link usa-social-link--facebook"
+                href="#">
+                <span>Facebook</span>
+              </a>,
+              <a
+                key="twitter"
+                className="usa-social-link usa-social-link--twitter"
+                href="#">
+                <span>Twitter</span>
+              </a>,
+              <a
+                key="youtube"
+                className="usa-social-link usa-social-link--youtube"
+                href="#">
+                <span>YouTube</span>
+              </a>,
+              <a
+                key="rss"
+                className="usa-social-link usa-social-link--rss"
+                href="#">
+                <span>RSS</span>
+              </a>,
+            ]}
+          />
+          <h3 className="usa-footer__contact-heading">Agency Contact Center</h3>
+          <Address
+            medium
+            items={[
+              <a key="telephone" href="tel:1-800-555-5555">
+                (800) CALL-GOVT
+              </a>,
+              <a key="email" href="mailto:info@agency.gov">
+                info@agency.gov
+              </a>,
+            ]}
+          />
+        </div>
+      </div>
+    }
+  />
+)
+
+export const BigFooter = (): React.ReactElement => (
+  <Footer
+    big
+    returnToTop={returnToTop}
+    primary={
+      <div className="grid-container">
+        <div className="grid-row grid-gap">
+          <div className="tablet:grid-col-8">
+            <FooterNav
+              big
+              links={[
+                [
+                  <h4 key="1" className="usa-footer__primary-link">
+                    Topic
+                  </h4>,
+                  ...Array(2).fill(
+                    <a className="usa-footer__secondary-link" href="#">
```

Comment:
I realize this is inconsistent with the primary links in the slim/medium footer, but according to the markup at https://designsystem.digital.gov/components/footer/ it looks like the `usa-footer__secondary-link` class is not on the `<a>` tags at all (just on the enclosing `<li>`). I wonder if this will resolve the issue you were having with the hover styles?

Reply:
yay nice catch --  thank you 🔍 .  Yes strange its not consistent.   

Still doesn't fix the hover styles thing though.  I keep seeing the blue color in the styles being applied by `a:hover {
    color: #1a4480;
}` coming from `typography.scss`. I'll keep looking around but I may ask to pair later today.

Reply:
This is now documented in #191